src/lib/cameraControls.ts

```diff
@@ -1,4 +1,12 @@
 import { MouseControlType } from 'wasm-lib/kcl/bindings/MouseControlType'
+import { platform } from './utils'
+
+const PLATFORM = platform()
+const META =
+  PLATFORM === 'macos' ? 'Cmd' : PLATFORM === 'windows' ? 'Win' : 'Super'
+const ALT = PLATFORM === 'macos' ? 'Option' : 'Alt'
+// Apple mouse scrolls without a wheel.
+const SCROLL = PLATFORM === 'macos' ? 'Scroll' : 'Scroll wheel'
```

Comment:
Now that you're mentioning it, "Scroll wheel" assumes a mouse with a wheel for scrolling anyway, so shouldn't we use "Scroll" as a more generic term everywhere, not just MacOS? I'm thinking a Microsoft Surface, a computer with a space mouse, etc.

Reply:
I'd be happy to nuke "wheel".